packages/config/src/layer2s/fraxtal.ts

```diff
@@ -1,17 +1,62 @@
 import { EthereumAddress, UnixTime } from '@l2beat/shared-pure'
 
-import { underReview } from './templates/underReview'
+import { ProjectDiscovery } from '../discovery/ProjectDiscovery'
+import { opStack } from './templates/opStack'
 import { Layer2 } from './types'
 
-export const fraxtal: Layer2 = underReview({
-  id: 'fraxtal',
+const discovery = new ProjectDiscovery('fraxtal')
+
+const upgradeability = {
+  upgradableBy: ['ProxyAdmin'],
+  upgradeDelay: 'No delay',
+}
+
+export const fraxtal: Layer2 = opStack({
+  daProvider: {
+    name: 'External',
+    riskView: {
+      value: 'External - Fraxtal Custom Module',
+      description:
+        'Fraxtal uses a separate data availability module developed by the Frax Core Team. The data needed for proof construction is not published on-chain.',
```

Comment:
```suggestion
        'Fraxtal uses a separate data availability module developed by the Frax Core Team. The data needed for proof construction is not published onchain.',
```

just a style nitpick

Reply:
Thanks, updated